functions/supplier/sftp-supplier/src/main/java/org/springframework/cloud/fn/supplier/sftp/SftpSupplierProperties.java

```diff
@@ -75,6 +74,11 @@
 	 */
 	private boolean deleteRemoteFiles = false;
 
+	/**
+	 * A SpEL expression resolving to the new name remote files must be renamed to after successful transfer.
+	 */
+	private Expression renameRemoteFilesTo = null;
```

Comment:
I think this is "per file", so better to call the property as `renameRemoteFileTo` and also mention a single file in the JavaDoc for this property.

Reply:
I have considered that, but then I preferred being consistent with `deleteRemoteFiles` naming convention and wrote the JavaDoc that way on purpose. Any thoughts on that?

Reply:
Right. I also would just call that `deleteRemoteFile`, but the bird is out of cage already.
Plus that one is definitely for all unconditionally, when "rename" is per file.
But yeah: that's just my opinion - no strict requirements.